cmd/support_archive/builder.go

```diff
@@ -168,12 +171,18 @@ func (builder CommandBuilder) runCollectors(log logd.Logger, supportArchive arch
 	pods := clientSet.CoreV1().Pods(namespaceFlagValue)
 	appName := getAppNameLabel(ctx, pods)
 
+	supportabilityEnabled, err := isSupportabilityEnabled(ctx, clientSet)
+	if err != nil {
+		return err
+	}
+
 	logInfof(log, "%s=%s", labels.AppNameLabel, appName)
 
 	fileSize := loadsimFileSizeFlagValue * Mebi
 	collectors := []collector{
 		newOperatorVersionCollector(log, supportArchive),
-		newLogCollector(ctx, log, supportArchive, pods, appName, collectManagedLogsFlagValue),
+		newLogCollector(ctx, log, supportArchive, pods, appName, collectManagedLogsFlagValue, supportabilityEnabled),
```

Comment:
Do I get this correct that we decided to only allow log collection of operator pods if the flag is enabled?

Reply:
True. Required permissions for supportability moved into separate role.